### ISSUE TITLE
add toggle-able collapse button for modules 

### ISSUE THREAD

**Issue by aashna27** [publiclab/image-sequencer#569]
This is a sub-issue of #487.
### Add a toggle button next to module name to hide image and description, leaving just the module name and line.

**Comment by aashna27** [comment]
I would like to work on this.

**Comment by tech4GT** [comment]
Aashna, didn't this get solved in some pr, I remember reviewing something similar, can you possibly check for this?

**Comment by aashna27** [comment]
I had absolutely no idea, I ll check for it.

**Comment by aashna27** [comment]
> Aashna, didn't this get solved in some pr, I remember reviewing something similar, can you possibly check for this?

yep that's correct 😆  https://github.com/publiclab/image-sequencer/pull/534 so would you like me to close my pr ?
